target/linux/realtek/files-5.10/include/dt-bindings/clock/rtl83xx-clk.h

```diff
@@ -7,9 +7,10 @@
 #ifndef __DT_BINDINGS_CLOCK_RTL83XX_H
 #define __DT_BINDINGS_CLOCK_RTL83XX_H
 
-#define CLK_CPU		0
-#define CLK_MEM		1
-#define CLK_LXB		2
-#define CLK_COUNT	3
+#define CLK_SW		0
+#define CLK_CPU		1
+#define CLK_MEM		2
+#define CLK_LXB		3
+#define CLK_COUNT	4
```

Comment:
Why are you changing the values of these `#define`s? Adding `#define CLK_SWITCH 4` would be a smaller change, and the values are arbitrary anyway.